packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -21,3 +21,14 @@ export const sync = () => ({
       'Apply synchronization when there is a 100% match between organization and snapshot resources.',
   }),
 });
+
+export const previewLevel = () => ({
+  previewLevel: flags.enum({
+    char: 'p',
+    description:
+      'Whether to only display a light or detailed preview of the overall changes',
+    options: ['light', 'detailed'],
```

Comment:
Store the value of the enum in an actual enum and use Object.keys ;) (or value I don't remember tbh ;)

You have examples of that I think with the resources coming from `platform-client` :)